Hints.py

```diff
@@ -114,73 +114,80 @@ def isRestrictedDungeonItem(dungeon, item):
     return False
 
 
-def add_hint(spoiler, world, IDs, gossip_text, count, location=None, force_reachable=False):
-    random.shuffle(IDs)
+def add_hint(spoiler, world, groups, gossip_text, count, location=None, force_reachable=False):
+    random.shuffle(groups)
     skipped_ids = []
```

Comment:
should rename this too?

Reply:
ah yeah I guess I missed some. this PR was more messy than I hoped